.github/actions/prepare-frontend/action.yml

```diff
@@ -7,13 +7,21 @@ runs:
       with:
         node-version: 14.x
         cache: 'yarn'
+    - name: Check to see if dependencies should be cached
+      if: ${{ contains(github.event.head_commit.message, '[ci nocache]') }}
+      run: |
+        echo "Commit message includes [ci nocache]; dependencies will NOT be cached"
+        yarn cache clean
```

Comment:
Should we clean, or just skip as the PR title says ? (this is an odd to the previous comment. Or one, or another should take place)